src/dispatch/static/dispatch/src/auth/editSheet.vue

```diff
@@ -0,0 +1,69 @@
+<template>

```

Comment:
Do we prevent non-admins to change users' roles?

Reply:
Not yet, will land that in another PR.